### PR TITLE
Unified CI

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
C.I. is now controlled by a minimal ci.yml and generated with https://github.com/NullVoxPopuli/ember-ci-update

### CODE REVIEW

File: ci.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+provider: github
```
Comment (NullVoxPopuli) [inline ci.yml:1]: This one file now generates all of the github actions workflow

File: package.json [hunk 2]
```diff
@@ -8,8 +8,11 @@
     "dev:ember": "pnpm run --filter test-app start --port 0",
     "dev:addon": "pnpm run --filter ember-aria-utilities start --no-watch.clearScreen",
     "dev:docs": "pnpm run --filter docs start",
-    "build": "pnpm run --filter ember-resources build",
```
Comment (NullVoxPopuli) [inline package.json:11]: updated scripts from the last PR

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,27 +1,14 @@
 name: CI
-on:
+"on":
```
Comment (kaitlinmuth) [inline .github/workflows/ci.yml:2]: Weird that this needs to be in quotes when the other keys don't

File: ember-aria-utilities/package.json [hunk 4]
```diff
@@ -85,7 +85,7 @@
     "rollup-plugin-ts": "2.0.7",
     "semantic-release": "^19.0.2",
     "prettier": "^2.0.0",
```
Comment (kaitlinmuth) [inline ember-aria-utilities/package.json:87]: Are we electing not to pin dependencies on our open source stuff?

**Review by kaitlinmuth — APPROVED** [review]

File: ember-aria-utilities/package.json (hunk 4 above)
Comment (NullVoxPopuli) [reply ember-aria-utilities/package.json:87]: correct -- this allows more aggressive "auto-maintenance" while also ensuring wider range of compatibility with whatever we use internally.

given a wide range of `^4.0.0`, internally, we can be on "whatever version", and to bump that (if it were a dependency version), would be a breaking change, as we'd be declaring dropped support for `4.0.0`.

additionally, we have extra re-assurance of non-breakage, due to more robust C.I.:  https://github.com/CrowdStrike/ember-aria/actions/runs/2416524904

specifically, we have many "scenarios" which are testing supported ember versions and typescript versions, and then a test that deletes the whole lockfile and re-generates it (the lockfile is used strictly in all C.I. jobs). 
Allowing a floating-deps test allows us to catch potential configuration issues and bugs in the broader JS ecosystem before they'd trickle into our internal code / updates.

File: .github/workflows/ci.yml (hunk 3 above)
Comment (vitch) [reply .github/workflows/ci.yml:2]: Is it a keyword or something?

**Review by vitch — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (NullVoxPopuli) [reply .github/workflows/ci.yml:2]: not that I've been able to find :thinking:

**Pull request merged**